_posts/2021-11-08-todo.markdown

```diff
@@ -0,0 +1,290 @@
+---
+layout: post
+title: "TODO come up with a title"
+date: 2021-11-08 03:44:05 +0100
+comments: true
+published: true
+categories: ["post"]
+tags: ["General", ".NET", "Travel"", "Startup"]
+author: Maarten Balliauw
+---
+
+For close to two years now, I've had [SpeakerTravel](https://speaker.travel/) online. It's a tool that helps conference organizers to book flights for speakers. You invite speakers, they pick their flight of choice (within a budget the organizer can specify), and the organizer can then approve and book the flight with a single click.
+
+In this post, I want to go a bit into the process of building this tool. Background, the technical side, the business side, and maybe a follow-up post that covers any questions you may have after reading.
+
+{% include toc %}
+
+There's also a table of contents, so brace yourself for a longread!
+
+## Background
+
+Before COVID threw a wrench in offline activities, our [user group](https://www.azug.be/) was organizing [CloudBrew](https://www.cloudbrew.be/), a 2-day conference with speakers from across Europe.
```

Comment:
Sometimes had speakers from US as well, but not so relevant I guess

Reply:
Rewriting a bit, thanks!